packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -23,6 +23,26 @@ v0.118.0
 - Refactor ErrorDialog to use `title` prop in `f-mega-modal`
 
 
+v0.119.0
+------------------------------
+*May 18, 2021*
+
+### Added
+- Ability to handle orders with the service type `dinein`
+- Table number/name input field
+
+
+v0.118.0
```

Comment:
that's strange that this is in your PR as a change. Might be worth rebasing?

Reply:
Strange indeed. I've sorted that now.